packages/synapse-react-client/package.json

```diff
@@ -59,7 +59,8 @@
     "@tanstack/query-core": "5.22.2",
     "@tanstack/react-query": "5.22.2",
     "@tanstack/react-query-devtools": "5.24.0",
-    "@tanstack/react-table": "^8.20.5",
+    "@tanstack/react-table": "^8.20.6",
+    "@tanstack/react-virtual": "^3.11.1",
```

Comment:
@tanstack/react-virtual is very small, and was easy to figure out how to get it to work with @tanstack/react-table compared to react-window, since examples exist in the react-table docs.

We're also removing another virtualization library embedded in react-base-table, so while I don't _want_ to have multiple dependencies that do the same thing, I think this is progress.